src/content/pages/team.html

```diff
@@ -119,6 +119,75 @@ <h2 class="section-title">{{ site.team_page.committers.title }}</h2>
     </div>
 </section>
 
+<section class="section">
```

Comment:
For each of these sections, I think we should check to make sure there is at least one member before rendering the section.

Reply:
@nzakas Done, I added a check - `{% if team.teamName.length > 0 %}`